core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -132,7 +132,7 @@ class BrokerTopicMetrics(name: Option[String]) extends KafkaMetricsGroup {
   }
 }
 
-object BrokerTopicStats extends Logging {
+object BrokerTopicStats {
```

Comment:
The changes are mechanical,  the diffstat is +483 −315 (which is manageable) and there's a separate commit for the refactoring. So, I think we don't need a separate PR. Let's see what @junrao says.